sda/cmd/mapper/mapper.go

```diff
@@ -102,14 +103,14 @@ func main() {
 				}
 
 				for _, aID := range mappings.AccessionIDs {
-					log.Debugf("Mapped file to dataset (corr-id: %s, datasetid: %s, accessionid: %s)", delivered.CorrelationId, mappings.DatasetID, aID)
```

Comment:
why remove these logs?

Reply:
By accident :face_with_peeking_eye: . Good that you noticed. Readded in e263eb3.